sprints/2024/02_08/ztf_bts-ngc.ipynb

```diff
@@ -3,9 +3,7 @@
   {
```

Comment:
<p>Should we maybe cite original code, <a href="https://spacetelescope.github.io/mast_notebooks/notebooks/PanSTARRS/PS1_image/PS1_image.html" target="_blank">https://spacetelescope.github.io/mast_notebooks/notebooks/PanSTARRS/PS1_image/PS1_image.html</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/notebooks_lf/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>